packages/datastore/src/datastore/datastore.ts

```diff
@@ -360,6 +364,20 @@ const checkSchemaInitialized = () => {
 	}
 };
 
+/**
+ * Throws an excpetion if the schema is using a pragma that is not supported.
+ *
+ * @param pragma schema pragma
+ */
+const checkSchemaPragma = (pragma: string) => {
+	const supportedRange = '^3.1.0';
```

Comment:
Just a random number here now.

Reply:
Is there a meaningful range we can put here before merging? If we don't know yet, can you add an internal task to update this when the CLI update is released?

Reply:
`3.1.0` is the latest version so `3.2.0` would be the lowest possible. I'll add a todo and a task.